docs/dev-guide/getting-started.md

```diff
@@ -30,7 +30,46 @@ Download [Eclipse IDE for Enterprise Java Developers](https://www.eclipse.org/do
 
 Git fork and clone the quality measure and cohort engine repository using GitHub Standard Fork & Pull Request Workflow. For instance, ``git clone git@github.com:<myuser>/Alvearie/quality-measure-and-cohort-service.git`` where ``myuser`` is your GitHub user or GitHub organization.
 
-Once cloned locally, execute the following from the root of the repository:
+The code in the quality-measure-and-cohort-service project has dependencies on code in the https://github.com/Alvearie/rest-service-framework project. In order to build, we must set up access to the snapshot packages in the rest-service-framework project. To do this, you must first create a Personal Access Token by going to https://github.com/settings/tokens and logging in with your github userid. Click "Personal access tokens" in the left side menu, then click the "Generate new token" button. Under the "Select Scopes," check read:packages and click the "Generate Token" button. Copy the generated token string as it will be used in the settings.xml updates below.
```

Comment:
It appears that the github actions are working for your branch, but it isn't clear to me how without some additional work to enable these settings. Have you thought through that interaction?

Reply:
Using GitHub Actions uses a GITHUB_TOKEN which is created automatically by GitHub. I added a reference to the build.yml to make sure it can use it and authenticate.